config.lua

```diff
@@ -16,7 +16,7 @@ Config.HandcuffTimer              = 15 * 60000 -- 15 mins
 Config.EnableJobBlip              = true -- enable blips for cops on duty, requires esx_society
 Config.EnableCustomPeds           = true -- enable custom peds in cloak room? See Config.CustomPeds below to customize peds
 
-Config.EnableESXService           = true -- enable esx service?
+Config.EnableESXService           = false -- enable esx service?
```

Comment:
disabling this will affect other scripts that rely on counting how many police are in active duty.